### ISSUE TITLE
JSX attribute completion runs only once for a tag

### ISSUE THREAD

**Issue by brerdem** [microsoft/TypeScript#52634]
Does this issue occur when all extensions are disabled?: Yes/No
Yes

- VS Code Version: 1.75.0 Universal
- OS Version: macOS Monterey 12.3

Steps to Reproduce:

1. Set JSX Attribute Completion setting to "braces"
2. Add a JSX tag or component
3. Write first letters of the attribute, the suggestion box appears. Hit Tab/Enter
4. VS Code adds ={} for the first attribute, however for the consecutive attributes does nothing.

![recording](https://user-images.githubusercontent.com/778976/216991879-08cffe02-6298-426e-ac1d-f3a56d700bbd.gif)

**Comment by gabritto** [comment]
@mjbvz were you able to repro this? I just tried and couldn't repro with the latest TS (built from current `main`).
Maybe I'm missing other relevant JSX/TSX preferences or something in a tsconfig? @brerdem do you know what version of TypeScript vscode was using when you ran into this issue?

**Comment by mjbvz** [comment]
No I can't repo this either: 

![Screenshot 2023-02-09 at 12 19 51 PM](https://user-images.githubusercontent.com/12821956/217927942-78180b97-53a7-4001-ab3a-08f7beaaf7a3.png)

**Comment by brerdem** [comment]
@gabritto It's a bare Vite project using typescript `^4.9.3`

And here is the tsconfig file 

```{
  "compilerOptions": {
    "target": "ESNext",
    "useDefineForClassFields": true,
    "lib": ["DOM", "DOM.Iterable", "ESNext"],
    "allowJs": false,
    "skipLibCheck": true,
    "esModuleInterop": false,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "module": "ESNext",
    "moduleResolution": "Node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx"
  },
  "include": ["src"],
  "references": [{ "path": "./tsconfig.node.json" }]
}
```

I also reset the settings and disabled extensions completely, even re-installed VS Code but had no luck.

I also thought maybe the problem happens on certain attributes but it seems no matter what attribute is, the completion works only for the first one. 

![ss](https://user-images.githubusercontent.com/778976/218088730-b428e984-c2a2-487a-8e8c-049a5a961cc4.gif)

**Comment by gabritto** [comment]
@brerdem thank you for the information. I managed to reproduce this bug with TypeScript 4.9.5.
This is a duplicate of https://github.com/microsoft/TypeScript/issues/51845, which has been fixed by https://github.com/microsoft/TypeScript/pull/51855 in the latest TypeScript version (5.0 beta).

**Comment by brerdem** [comment]
Thanks @gabritto. I'd like to inform that the problem persists in the nightly build I installed which is `5.0.0-dev.20230210`

**Comment by brerdem** [comment]
@gabritto I'm not 100% sure but this may not be a TypeScript related issue. I have opened very same project with WebStorm and completion works just fine.

![ss3](https://user-images.githubusercontent.com/778976/218269526-6b5112d0-c786-48ef-b440-e493af39fbf1.gif)
